development/roles.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Roles
+
+pagination:
+  next:
+    link: ../herramientas
+    name: Herramientas
```

Comment:
The `previous` pagination should also be placed here for a better navigation

Reply:
I'll add it.